boards/nxp/frdm_mcxn947/doc/index.rst

```diff
@@ -82,6 +82,8 @@ The FRDM-MCXN947 board configuration supports the following hardware features:
 +-----------+------------+-------------------------------------+
 | CTIMER    | on-chip    | counter                             |
 +-----------+------------+-------------------------------------+
+| LPCMP     | on-chip    | low power analog comparator         |
```

Comment:
typically the 3rd column maps to a zephyr interface if it can, might make sense to say something like "sensor (comparator)"